internal/generate/utils.go

```diff
@@ -265,3 +268,10 @@ func allItemsAreSame[T comparable](a []T) bool {
 	}
 	return true
 }
+
+// sortedKeys returns a []string of sorted keys from a map. Used to ensure deterministic ordering of generated code.
+func sortedKeys[T any](m map[string]T) []string {
+	keys := slices.Collect(maps.Keys(m))
+	sort.Strings(keys)
+	return keys
```

Comment:
You can update this body to be the following. Alternatively, you can eliminate this function and use this at each call site. Either way works for me so I'll approve this and defer to you!

```suggestion
	keys := slices.Sorted(maps.Keys(m))
	return keys
```